templates/municipal_chief/municipal/running_mate/detail.html.twig

```diff
@@ -0,0 +1,33 @@
+{% extends 'municipal_chief/_layout.html.twig' %}
+
+{% block municipal_chief_content %}
+    <a href="{{ path('app_municipalchief_municipal_runningmaterequest') }}" class="back-to-list icon--with-text b__nudge--top-15 b__nudge--bottom">
+        {{ include('components/caret--left.html.twig') }}
+        Retour à la liste des candidats colistiers
+    </a>
+
+    <hr />
+
+    <div class="l__wrapper--slim">
```

Comment:
it seems to be duplicated from `running_mate/detail.html.twig`, could you use a partial template for this part, please?